static/js/modules/tables.js

```diff
@@ -12,11 +10,30 @@ require('datatables');
 require('drmonty-datatables-responsive');
 
 var helpers = require('./helpers');
+var download = require('./download');
 var analytics = require('./analytics');
 
 var hideNullTemplate = require('../../templates/tables/hideNull.hbs');
+var exportWidgetTemplate = require('../../templates/tables/exportWidget.hbs');
 
 var simpleDOM = 't<"results-info"ip>';
+var browseDOM = '<"js-results-info results-info results-info--top"pilfr>' +
+                '<"panel__main"t>' +
+                '<"results-info"ip>';
+
+var DOWNLOAD_CAP = 100000;
+
+var CAP_EXCEEDED =
+    'Exports are limited to ' +
+    DOWNLOAD_CAP +
```

Comment:
Just noticed that this needs to be formatted (it was previously formatted with the Handlebars helper).